src/state/stake/stake-constants.ts

```diff
@@ -683,6 +684,15 @@ const AURORA_POOLS: StakingTri[] = [
     rewarderAddress: '0xEfA0D8957101052D356DaDF4fa666b4AcC74b407',
     allocPoint: 1,
     poolType: PoolType.DUAL_REWARDS
+  }),
+  createMCV2Pool({
+    ID: 50,
+    poolId: 43,
+    tokens: [STAUR[ChainId.AURORA], AURORA[ChainId.AURORA]],
+    lpAddress: '0x22E38A04c14624e6deb66762Dc98E793db2C80Cb',
+    rewarderAddress: '0xB247fA13a9BE90123C4B94982A2b30c60E6182ab',
+    allocPoint: 1,
+    poolType: PoolType.DUAL_REWARDS
```

Comment:
You're gonna hate me but can we introduce another PoolType for maintainability

You don't know what you don't know in the future lol